frontend/src/components/chatInput/ChatInput.styled.ts

```diff
@@ -2,52 +2,49 @@ import styled from "styled-components";
 import { BsImage } from "react-icons/bs";
 
 const InputWrapper = styled.div`
```

Comment:
blank spaces in css is good idea, this split the whole props to several parts of it, and when e.g. i read it, it's easier, if prettier do it for u, try to change it, but on some SC components u don't remove this blank spaces, i think it's your work, not prettier ;)